week-2/Homework/mandatory/1-fix-functions.js

```diff
@@ -13,9 +13,9 @@ function mood() {
 
 function greaterThan10() {
   let num = 10;
-  let isBigEnough;
+//  let isBigEnough;
 
-  if (isBigEnough) {
+  if (num>=10) {//condition whether num is greater or equals
```

Comment:
Nice! This is good! If you were to use the `isBigEnough` variable, how would you do it?